addons/mail/models/mail_followers.py

```diff
@@ -115,7 +115,7 @@ def _get_recipient_data(self, records, subtype_id, pids=None, cids=None):
         users.notification_type AS notif, array_agg(groups.id) AS groups
     FROM res_partner partner
     LEFT JOIN res_users users ON users.partner_id = partner.id
```

Comment:
filter on active users should actually be made on this relation
```suggestion
     LEFT JOIN res_users users ON users.partner_id = partner.id AND users.active
```